cpp/tests/test_analyze_result.cpp

```diff
@@ -468,6 +469,89 @@ TEST(TestEnsembleParamsPercentile, basic)
               14);
 }
 
+TEST(TestEnsembleParamsPercentile, abm_basic)
```

Comment:
I think it is mostly used to test `ensemble_params_percentile` function but not sure it will produce a graph result. I made the name change for now though.